### PR TITLE
Bump RTE to v0.2.7

### PR DESCRIPTION
**Pull request by Tal-or**
Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

**Comment by Tal-or** [comment]
@fromanirh we also need a new tag that the numaresources-operator will be able to refer to.

**Comment by fromanirh** [comment]
we can jump to RTE v0.2.7

**Comment by Tal-or** [comment]
> we can jump to RTE v0.2.7

The CI will probably fail, we need to fix the ```E2E_TOPOLOGY_MANAGER_POLICY=single-numa-node``` in
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-kni/resource-topology-exporter/openshift-kni-resource-topology-exporter-master.yaml#L47

**Comment by fromanirh** [comment]
> > we can jump to RTE v0.2.7
> 
> The CI will probably fail, we need to fix the `E2E_TOPOLOGY_MANAGER_POLICY=single-numa-node` in https://github.com/openshift/release/blob/master/ci-operator/config/openshift-kni/resource-topology-exporter/openshift-kni-resource-topology-exporter-master.yaml#L47

good point. Let's take this chance to review this flow. Like, adding a helper in this repo so changes like this one don't impact the `openshift/release` repo anymore.

**Comment by Tal-or** [comment]
WIP. Further changes are needed. will update the PR soon.

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/jaypipes/ghw v0.8.1-0.20210609141030-acb1a36eaf89
 	github.com/jaypipes/pcidb v0.6.0
-	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.2.6-0.20211104082133-eabd0b87af8c
+	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.2.7-0.20211115173956-714a94763db8
```
Comment (fromanirh) [inline go.mod:8]: please use `v0.2.7`

**Comment by Tal-or** [comment]
/retest
Looks like flakiness

**Review by fromanirh — APPROVED** [review]
/lgtm

**Comment by Tal-or** [comment]
/retest
Again flakiness:
```
• Failure [7.604 seconds]
[RTE][InfraConsuming] Resource topology exporter
/go/src/github.com/openshift-kni/resource-topology-exporter/vendor/github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/rte/rte.go:45
  with cluster configured
  /go/src/github.com/openshift-kni/resource-topology-exporter/vendor/github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/rte/rte.go:84
    it should react to pod changes using the smart poller [It]
    /go/src/github.com/openshift-kni/resource-topology-exporter/vendor/github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/rte/rte.go:85
    Timed out after 5.000s.
      
    didn't get updated node topology info
    Expected
        <bool>: false
    to be true
    /go/src/github.com/openshift-kni/resource-topology-exporter/vendor/github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/rte/rte.go:133
```
We'll probably skip this test if it'll failed again.
We're about to move to the notification file in the future anyway

**Pull request merged**